test/integration/metrics_health_test.lua

```diff
@@ -58,6 +66,29 @@ g.test_metrics_health_handler = function()
     t.assert_equals(resp.status, 200)
 end
 
+g.test_metrics_custom_is_health_handler = function()
+    local main_server = g.cluster:server('main')
+    main_server:exec(function()
+        local metrics = require('cartridge.roles.metrics')
+        metrics.set_custom_is_health_handler(function(req)
+            local health = require('cartridge.health')
+            local resp = req:render{
+                json = {
+                    my_healthcheck_format = health.is_healthy()
+                }
+            }
+            resp.status = 200
+            return resp
+        end)
+    end)
+
+    helpers.upload_default_metrics_config(g.cluster)
```

Comment:
Should this one work if I switch handle setup and configuration update?

Reply:
Yes. `set_is_health_handler` and `upload_default_metrics_config` work independently.